packages/yoastseo/src/worker/bootstrapAnalysis.js

```diff
@@ -6,14 +6,10 @@ self.onmessage = ( { data } ) => {
 		return;
 	}
 
-	for ( const dependency in data.dependencies ) {
-		if ( ! Object.prototype.hasOwnProperty.call( data.dependencies, dependency ) ) {
-			continue;
-		}
-
-		self.importScripts( data.dependencies[ dependency ] );
+	for ( const dependency of data.dependencies ) {
+		self.importScripts( dependency );
 
-		if ( dependency === "lodash" ) {
+		if ( dependency.includes( "lodash" ) ) {
```

Comment:
Why is this no longer a precise match? Is it now `vendor/lodash`? If so, maybe a little more precise `endsWith` instead? Or is that a `lodash-es` thing?